extensions/amp-video-iframe/0.1/amp-video-iframe.js

```diff
@@ -266,6 +268,22 @@ class AmpVideoIframe extends AMP.BaseElement {
       return;
     }
 
+    if (eventReceived == 'analytics') {
+      const analyticsEventType = data['eventType'];
+
+      user().assertString(
+          analyticsEventType, '`eventType` missing in analytics event');
+
+      user().assert(
+          analyticsEventType != VideoAnalyticsEvents.CUSTOM,
```

Comment:
Why do you restrict this? Is there a danger of an infinite loop?

Reply:
Not infinite, but being caught twice and misreporting on the second call.